R-package/src/install.libs.R

```diff
@@ -56,30 +56,27 @@ if (!use_precompile) {
   # Install
   system(paste0(cmake_cmd, " .."))
   system(build_cmd)
-  dest <- file.path(R_PACKAGE_DIR, paste0("libs", R_ARCH))
-  dir.create(dest, recursive = TRUE, showWarnings = FALSE)
   src <- paste0(lib_folder, "/lib_lightgbm", SHLIB_EXT)
   
 } else {
   
-  lib_folder <- paste0(R_PACKAGE_SOURCE, "/src")
-  print(lib_folder)
-  dest <- file.path(R_PACKAGE_DIR, paste0("libs", R_ARCH))
-  dir.create(dest, recursive = TRUE, showWarnings = FALSE)
-  print(dest)
-  if (file.exists(paste0(lib_folder, "/../../lib_lightgbm", SHLIB_EXT))) {
-    src <- paste0(lib_folder, "/../../lib_lightgbm", SHLIB_EXT)
+  lib_folder <- paste0(R_PACKAGE_SOURCE, "/..")
+  if (file.exists(paste0(lib_folder, "./lib_lightgbm", SHLIB_EXT))) {
```

Comment:
... hmm, the path will be ```/.../lib_lightgbm``` here ? 
We think we can unify all variables of ```folder``` end with ```/``` to avoid the confusing .